ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -320,16 +321,24 @@ protected EngineStatus getInvalidBlockHashStatus() {
   }
 
   private void logImportedBlockInfo(final Block block, final double timeInS) {
-    LOG.info(
-        String.format(
-            "Imported #%,d / %d tx / base fee %s / %,d (%01.1f%%) gas / (%s) in %01.3fs. Peers: %d",
-            block.getHeader().getNumber(),
-            block.getBody().getTransactions().size(),
+    final StringBuilder message = new StringBuilder();
```

Comment:
Why not add it all the time and just have "N/A" if it's not present?  This is what happened with BaseFee.  Then we can keep the log line in one string, which will increase readability.

Reply:
For non-mainnet cases, there would always be an irrelevant portion of log and I think there's value in keeping this printout as small as possible for readability in the log. 
IMO baseFee should be optional too, although it's tied to the london fee market which private networks might still use.

I am not opposed to favouring dev readability instead, if anyone decided to open another PR to change this.